requirements/edx/github.txt

```diff
@@ -107,3 +107,6 @@ git+https://github.com/edx/edx-proctoring.git@1.3.0#egg=edx-proctoring==1.3.0
 git+https://github.com/mitodl/edx-sga.git@d019b8a050c056db535e3ff13c93096145a932de#egg=edx-sga==0.7.1
 git+https://github.com/open-craft/xblock-poll@7ba819b968fe8faddb78bb22e1fe7637005eb414#egg=xblock-poll==1.2.7
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.18#egg=xblock-drag-and-drop-v2==2.0.18
+# TODO: Deploy xblock-review to PyPI and pin it before going to master. Talk to Feanil if any questions
+# For the purpose of being able to build during this PR time, I'm setting the branch to PR. Will switch to PyPI later
+-e git+https://github.com/Dillon-Dumesnil/xblock-review@PR#egg=xblock-review
```

Comment:
If you put `==0.0` at the end of this line, the package will re-install everytime IIRC. Though - it might be reinstalling fine now for you.

Reply:
I think I'm going to not rock the boat on this one especially since it will be switched to the PyPI version as soon as this PR is good to go.